lib/i18n_routable/translation_assistant.rb

```diff
@@ -15,16 +15,18 @@ def untokenize_segment segment
 
     # /posts => ["/:i18n_posts", ["posts"]]
     def convert_path_to_localized_regexp path, requirements={}, anchor= true
-      spec = Journey::Parser.new.parse(path)
+      parser = ActionDispatch::Journey::Parser.new
+      spec = parser.parse(path)
       visitor = RouteTranslationVisitor.new
       new_path = visitor.accept(spec)
+      new_path_spec = parser.parse(new_path)
       new_requirements = requirements.merge(visitor.route_translations)
-      [new_path, new_requirements]
+      [new_path, new_path_spec, new_requirements]
```

Comment:
Technically, changing this method signature is a breaking change to the API. While I am sure this is necessary, we should call it out in documentation.